lib/AFS/CellCC/Util.pm

```diff
@@ -331,4 +332,58 @@ scratch_ok($$$$$) {
     return 1;
 }
 
+# Return the checksum of the file received as an argument.
+# The checksum returned by this function will be in following
+# format: "algorithm:checksum", where algorithm is the algorithm
+# of choice (e.g. MD5) and checksum is the checksum of the file
```

Comment:
Could you mention that the algorithm is one that is recognized by the Digest module? Just to be more specific about what we mean by the algorithm name.